app/models/booking.rb

```diff
@@ -1,2 +1,8 @@
 class Booking < ApplicationRecord
+  scope :find_room, ->(id){where room_id: id}
```

Comment:
```suggestion
  scope :by_room_ids, ->(ids){where room_id: ids}
```
nếu find thì nên xử lý find theo nhiều id nhé, vì nếu muốn lấy 1 thì find_by thôi là ổn

Reply:
e tìm theo 1 phòng thôi anh =))